### PR TITLE
Express_in

### PR DESCRIPTION
**Pull request by Basliel-Amsalu**
an adapter for extracting express in relationship between gene and ontology from gtex dataset

### CODE REVIEW

File: biocypher_metta/adapters/gtex_expression_adapter.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+                try:
+                    chr, pos = row[self.index["chr"]], row[self.index["pos"]]
+                    pos = int(pos)
+                    tissue = row[self.index["tissue"]].split(".")[0]
+                    gene_id = row[self.index["gene_id"]]
+                    ontology = self.gtex_tissue_ontology_map[tissue]
+                    if check_genomic_location(self.chr, self.start, self.end, chr, pos, pos):
+                        _source = gene_id
+                        _target = ontology
+                        _props = {}
```
Comment (dawit-melka) [inline biocypher_metta/adapters/gtex_expression_adapter.py:53]: could you add `p_value` as a property?

File: config/adapters_config_sample.yaml
Comment (dawit-melka) [inline config/adapters_config_sample.yaml:1]: It looks there a lot changes in this and other config files please review them and only commit necessary changes

**Review by dawit-melka — CHANGES_REQUESTED** [review]

File: config/schema_config.yaml [hunk 2]
```diff
@@ -732,6 +729,19 @@ gtex variant to gene expression association:
     source: str
     source_url: str
 
+gtex gene to ontology expression:
```
[the remark this answers is not in the document]
Comment (Basliel-Amsalu) [reply config/schema_config.yaml:732]: but the other one is for variant to gene association this is for gene and ontologie

[the remark this answers is not in the document]
Comment (dawit-melka) [reply config/schema_config.yaml:732]: have you checked this one? - [expressed in:](https://github.com/rejuve-bio/biocypher-kg/blob/31467c3598f6fa671dd41fa09a2e1bc7a36d2d20/config/schema_config.yaml#L464)

**Pull request merged**